modules/microshift-oc-mirror-transform-imageset-to-crio.adoc

```diff
@@ -51,7 +54,7 @@ yq '.spec.repositoryDigestMirrors[] as $item ireduce([]; . + [{"mirror": $item.m
       location = \"" + .mirror + "\"
       mirror-by-digest-only = true
       insecure = true
-      "' ./icsp.yaml
+      "' ./imagedigestmirrorset.yaml
```

Comment:
Now that we have ImageTagmirrorset.yaml i think cri-o might need to be configured with that too.

Reply:
I think we prefer digest references in MicroShift? I saw that comment in one of the older google docs if I remember correctly. I'm not certain if one is better than the other for microshift specifically.